stdlib/3/builtins.pyi

```diff
@@ -507,8 +507,14 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     def __init__(self, iterable: Iterable[Tuple[_KT, _VT]], **kwargs: _VT) -> None: ...
     def clear(self) -> None: ...
     def copy(self) -> Dict[_KT, _VT]: ...
-    def get(self, k: _KT, default: _VT = None) -> _VT: ...
-    def pop(self, k: _KT, default: _VT = None) -> _VT: ...
+    @overload
+    def get(self, k: _KT) -> Optional[_VT]: ...
+    @overload
+    def get(self, k: _KT, default: _T = ...) -> Union[_VT, _T]: ...
```

Comment:
This overload variant (and all the others like it) should no longer have a default `= ...`.